spec/controllers/api/event_hours_controller_spec.rb

```diff
@@ -53,6 +53,7 @@
   def expected_event_slots
     {
       'event_slot_id' => 1,
+      'capacity' => 5,
       'start_time' => '10 AM',
       'end_time' => '11 AM',
       'open_slots' => 5
```

Comment:
Suggestion: Consider adding reserved value to expected_event_slots, e.g., through factory and show open_slots expected results as capacity - reserved.